### PR TITLE
localization for principia(have some problem)

### PR DESCRIPTION
**Pull request by CindyRIng**
Every thing is done except this problem that _**some strings are hidden**_.
image

### CODE REVIEW

**Comment by pleroy** [comment]
[Automated message from GitHub Pull Request Builder]  Answer "ok to test" to trigger testing of this PR.

**Comment by CindyRIng** [comment]
ok to test

**Review by eggrobin — CHANGES_REQUESTED** [review]
Thank you very much for this significant contribution; this looks very good overall.

As I foretold I will have many comments about naming; let’s do this one window at a time, I commented on the strings referenced in `main_window.cs`, I will look at the other ones later.

I also took a quick look at the translations themselves, and I have a question about the best way to translate some of the more obscure technical terms we use.

File: Localization/en-us.cfg [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        #Principia_OBTAnalyser_Resault_AnomalisticPeriod = Anomalistic period
+        #Principia_OBTAnalyser_Resault_SemimajorAxis = Semimajor axis
+        #Principia_OBTAnalyser_Resault_Eccentricity = Eccentricity
+        #Principia_OBTAnalyser_Resault_Inclination = Inclination
+        #Principia_OBTAnalyser_Resault_LAN = Longitude of ascending node
+        #Principia_OBTAnalyser_Resault_NodalPrecession = Nodal precession
+        #Principia_OBTAnalyser_Resault_ArgumentPE = Argument of periapsis
+        #Principia_OBTAnalyser_Resault_ALTPE = Altitude of mean periapsis
+        #Principia_OBTAnalyser_Resault_ALTAP = Altitude of mean apoapsis
+        #Principia_OBTAnalyser_Plugin_Recurrence = Recurrence: <<1>>; <<2>>; <<3>> //<<1>> nuo <<2>> dto <<3>> cto
```
Comment (eggrobin) [inline Localization/en-us.cfg:94]: Keep the square brackets here, including in the Chinese version. [ν<sub>o</sub>; D<sub>T<sub>o</sub></sub>; C<sub>T<sub>o</sub></sub>] is a notation from [[Cap12]](https://github.com/mockingbirdnest/Principia/blob/b9981c096b5539fc0025a059c3230821a45d7880/documentation/bibliography.pdf), with a specific technical meaning; this is not just three numbers separated by semicolons.

File: ksp_plugin_adapter/main_window.cs [hunk 2]
```diff
@@ -354,9 +351,11 @@ internal class MainWindow : VesselSupervisedWindowRenderer {
     using (new UnityEngine.GUILayout.HorizontalScope()) {
       must_record_journal_ = UnityEngine.GUILayout.Toggle(
           value   : must_record_journal_,
-          text    : "Record journal (starts on load)");
+          text    : Localizer.Format("#Principia_MainMenu_LogText_Recordjournal"));
       UnityEngine.GUILayout.Label(
-          "Journaling is " + (journaling_ ? "ON" : "OFF"),
+          Localizer.Format("#Principia_MainMenu_LogText_RecordjournalResault",
```
Comment (eggrobin) [inline ksp_plugin_adapter/main_window.cs:356]: JournallingStatus.

File: ksp_plugin_adapter/main_window.cs [hunk 3]
```diff
@@ -354,9 +351,11 @@ internal class MainWindow : VesselSupervisedWindowRenderer {
...
+          (journaling_ ? Localizer.Format("#Principia_MainMenu_LogText_RecordjournalON") : 
+                         Localizer.Format("#Principia_MainMenu_LogText_RecordjournalOFF"))),
```
Comment (eggrobin) [inline ksp_plugin_adapter/main_window.cs:358]: JournallingStatus_ON/OFF.

File: Localization/zh-cn.cfg [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        #Principia_OBTAnalyser_OBTCategory_Supersynch = 超同步. 
+        #Principia_OBTAnalyser_Resault_OBTCategory =  <<2>><<1>>轨道 //<<1>>properties <<2>>primary.name
+        #Principia_OBTAnalyser_Resault_LowestAlt = 最低轨道高度
+        #Principia_OBTAnalyser_Resault_AltWarn1 = 发生撞击
+        #Principia_OBTAnalyser_Resault_AltWarn2 = 可能发生撞击
+        #Principia_OBTAnalyser_Resault_AltWarn3 = 发生再入
+        #Principia_OBTAnalyser_Resault_OBTElements = 平均轨道根数
+        #Principia_OBTAnalyser_Resault_SiderealPeriod = 经度平均周期
+        #Principia_OBTAnalyser_Resault_NodalPeriod = 纬度平均周期
+        #Principia_OBTAnalyser_Resault_AnomalisticPeriod = 近点平均周期
```
[the remark this answers is not in the document]
Comment (eggrobin) [reply Localization/zh-cn.cfg:85]: > this is indeed the mean anomalistic period

Actually on second thought « mean anomalistic period » feels wrong; there is only one kind of osculating period, as distinguishing the periods is something you can only do if you have some sort of perturbation theory—which largely amounts to having a mean element theory.

Let’s just use the names of the three kinds of periods without further qualification, thus
sidereal period: 恒星周期
anomalistic period: 近点周期
nodal period: 交点周期

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 5]
```diff
@@ -403,19 +414,19 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
                                      CelestialBody primary) {
     using (new UnityEngine.GUILayout.HorizontalScope()) {
       UnityEngine.GUILayout.Label(
-          $@"Recurrence: [{
+          $@"{Localizer.Format("#Principia_OBTAnalyser_StaticText_Recurrence")} [{
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:417]: `Principia_OrbitAnalyser_Recurrence_CapderouTriple`; make the triple a placeholder (`Recurrence: <<1>>`), and rewrite the call as follows:
```csharp
      string νₒ = recurrence?.nuo.ToString() ?? em_dash;
      string Dᴛₒ = recurrence?.dto.ToString("+0;-0") ?? em_dash;
      string Cᴛₒ = recurrence?.cto.ToString() ?? em_dash;
      UnityEngine.GUILayout.Label(
          Localizer.Format("#Principia_OrbitAnalyser_Recurrence_CapderouTriple",
                           $"[{νₒ}; {Dᴛₒ}; {Cᴛₒ}]"),
          GUILayoutWidth(8));
```

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 6]
```diff
@@ -403,19 +414,19 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
             recurrence?.nuo.ToString() ?? em_dash}; {
             recurrence?.dto.ToString("+0;-0") ?? em_dash}; {
             recurrence?.cto.ToString() ?? em_dash}]",
           GUILayoutWidth(8));
       UnityEngine.GUILayout.FlexibleSpace();
       autodetect_recurrence_ = UnityEngine.GUILayout.Toggle(
           autodetect_recurrence_,
-          "Auto-detect",
+          Localizer.Format("#Principia_OBTAnalyser_StaticText_AutoDetect"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:425]: Principia_OrbitAnalyser_Recurrence_AutoDetect

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 7]
```diff
@@ -403,19 +414,19 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
           UnityEngine.GUILayout.ExpandWidth(false));
     }
     using (new UnityEngine.GUILayout.HorizontalScope()) {
-      UnityEngine.GUILayout.Label("Cycle");
+      UnityEngine.GUILayout.Label(Localizer.Format("#Principia_OBTAnalyser_StaticText_Cycle"));
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:429]: Principia_OrbitAnalyser_Recurrence_Cycle

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 8]
```diff
@@ -427,7 +438,7 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
           revolutions_per_cycle > 0) {
         revolutions_per_cycle_ = revolutions_per_cycle;
       }
-      UnityEngine.GUILayout.Label("revolutions =",
+      UnityEngine.GUILayout.Label(Localizer.Format("#Principia_OBTAnalyser_StaticText_Revolutions"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:441]: Principia_OrbitAnalyser_Recurrence_Cycle_RevolutionsEquals

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 9]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
                                   UnityEngine.GUILayout.ExpandWidth(false));
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:455]: This needs to be translated too. Principia_OrbitAnalyser_Recurrence_Cycle_Days.

These are not days as in the 24-hour (or 6-hour on Kerbin) unit, which we write `d` or `d6`, a symbol which does not require translation.
These are « days » in a very strange technical sense, see https://github.com/mockingbirdnest/Principia/blob/15efd78a279d2504c30597b40d2bddd26fc9e06b/astronomy/orbit_recurrence.hpp#L29-L37.

It is best to use the word for « day » here ([[Cap12]](https://github.com/mockingbirdnest/Principia/blob/master/documentation/bibliography.pdf), which is in French, uses the word _jour_), thus here 天 in the zh-CN translation.

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 10]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
     }
     LabeledField(
-        "Subcycle",
+        Localizer.Format("#Principia_OBTAnalyser_Resault_Subcycle"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:458]: Principia_OrbitAnalyser_Recurrence_Subcycle

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 11]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
         $@"{recurrence?.subcycle.FormatN(0) ?? em_dash} days");
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:459]: This also needs to be translated; `Principia_OrbitAnalyser_Recurrence_SubcycleLengthInDays`, `<<1>> days`, `<<1>> 天`.

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 12]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
     LabeledField(
-        "Equatorial shift",
+        Localizer.Format("#Principia_OBTAnalyser_Resault_EquatorialShift"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:461]: `Principia_OrbitAnalyser_Recurrence_EquatorialShift`

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 13]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
         recurrence?.equatorial_shift.FormatEquatorialAngle(primary));
     LabeledField(
-        "Grid interval",
+        Localizer.Format("#Principia_OBTAnalyser_Resault_GridInterval"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:464]: `Principia_OrbitAnalyser_Recurrence_GridInterval`

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 14]
```diff
@@ -444,21 +455,21 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
         recurrence?.grid_interval.FormatEquatorialAngle(primary));
   }
 
   private void RenderOrbitGroundTrack(OrbitGroundTrack? ground_track,
                                       CelestialBody primary) {
     using (new UnityEngine.GUILayout.HorizontalScope()) {
       UnityEngine.GUILayout.Label(
-          "Longitudes of equatorial crossings of rev. #",
+          Localizer.Format("#Principia_OBTAnalyser_StaticText_REC"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:472]: `Principia_OrbitAnalyser_GroundTrack_LongitudesOfEquatorialCrossings`

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 15]
```diff
@@ -469,11 +480,11 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
     }
     var equatorial_crossings = ground_track?.equatorial_crossings;
     LabeledField(
-        "Ascending pass",
+        Localizer.Format("#Principia_OBTAnalyser_Resault_AscendingPass"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:483]: `Principia_OrbitAnalyser_GroundTrack_AscendingPass`

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 16]
```diff
@@ -469,11 +480,11 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
...
         equatorial_crossings?.longitudes_reduced_to_ascending_pass.
             FormatEquatorialAngleInterval(primary));
     LabeledField(
-        "Descending pass",
+        Localizer.Format("#Principia_OBTAnalyser_Resault_DescendingPass"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:487]: `Principia_OrbitAnalyser_GroundTrack_DescendingPass`

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 17]
```diff
@@ -493,7 +504,7 @@ internal abstract class OrbitAnalyser : VesselSupervisedWindowRenderer {
   private readonly PrincipiaPluginAdapter adapter_;
   private readonly DifferentialSlider mission_duration_ =
       new DifferentialSlider(
-          label            : "Duration",
+          label            : Localizer.Format("#Principia_OBTAnalyser_StaticText_Duration"),
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:507]: Principia_OrbitAnalyser_MissionDuration

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 18]
```diff
@@ -542,12 +553,12 @@ internal class CurrentOrbitAnalyser : OrbitAnalyser {
 
   protected override string ButtonText(string orbit_description) {
     return orbit_description == null
-        ? "Orbit analysis..."
-        : $"Analysis: {orbit_description}...";
+        ? Localizer.Format("#Principia_OBTAnalyser_ButtonOrbitAnalysis")
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:556]: Principia_CurrentOrbitAnalyser_ToggleButton

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 19]
```diff
@@ -542,12 +553,12 @@ internal class CurrentOrbitAnalyser : OrbitAnalyser {
...
+        : Localizer.Format("#Principia_OBTAnalyser_ButtonAnalysis", orbit_description);
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:557]: Principia_CurrentOrbitAnalyser_ToggleButtonWithDescription

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 20]
```diff
@@ -542,12 +553,12 @@ internal class CurrentOrbitAnalyser : OrbitAnalyser {
...
   }
 
   protected override string AnalysingText() {
-    return $"Analysing orbit of {predicted_vessel.vesselName}...";
+    return Localizer.Format("#Principia_OBTAnalyser_ButtonAnalysisOBTOf", predicted_vessel.vesselName);
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:561]: Principia_CurentOrbitAnalyser_Analysing

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 21]
```diff
@@ -572,12 +583,12 @@ internal class PlannedOrbitAnalyser : OrbitAnalyser {
 
   protected override string ButtonText(string orbit_description) {
     return orbit_description == null
-        ? "Analyse final trajectory..."
-        : $"Final orbit analysis: {orbit_description}...";
+        ? Localizer.Format("#Principia_OBTAnalyser_ButtonAnalyseFinalTR")
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:586]: Principia_PlannedOrbitAnalyser_ToggleButton

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 22]
```diff
@@ -572,12 +583,12 @@ internal class PlannedOrbitAnalyser : OrbitAnalyser {
...
+        : Localizer.Format("#Principia_OBTAnalyser_ButtonFinalOBTAnalysis", orbit_description);
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:587]: Principia_PlannedOrbitAnalyser_ToggleButtonWithDescription

File: ksp_plugin_adapter/orbit_analyser.cs [hunk 23]
```diff
@@ -572,12 +583,12 @@ internal class PlannedOrbitAnalyser : OrbitAnalyser {
...
   }
 
   protected override string AnalysingText() {
-    return $"Analysing final planned orbit of {predicted_vessel.vesselName}...";
+    return Localizer.Format("#Principia_OBTAnalyser_ButtonAnalysisFinalPlanObt", predicted_vessel.vesselName);
```
Comment (eggrobin) [inline ksp_plugin_adapter/orbit_analyser.cs:591]: Principia_PlannedOrbitAnalyser_Analysing

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 24]
```diff
@@ -109,28 +110,32 @@ public enum FrameType {
                             CelestialBody selected,
                             Vessel target_override) {
    if (target_override) {
-     return "Target Local Vert./Horiz. at " + selected.NameWithArticle();
+     return Localizer.Format("#Principia_RFS_DynamicTitle_Target", selected.NameWithArticle());
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:113]: Principia_ReferenceFrameSelector_Name_Target

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 25]
```diff
@@ -109,28 +110,32 @@ public enum FrameType {
...
    }
    switch (type) {
      case FrameType.BODY_CENTRED_NON_ROTATING:
-       return selected.name + "-Centred Inertial";
+       return Localizer.Format("#Principia_RFS_DynamicTitle_XCI",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:117]: Principia_ReferenceFrameSelector_Name_BodyCentredNonRotating

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 26]
```diff
@@ -109,28 +110,32 @@ public enum FrameType {
...
-       return selected.name + "-Centred Inertial";
+       return Localizer.Format("#Principia_RFS_DynamicTitle_XCI", 
+                                            selected.name);
      case FrameType.BARYCENTRIC_ROTATING:
         if (selected.is_root()) {
           throw Log.Fatal("Naming barycentric rotating frame of root body");
         } else {
-          return selected.referenceBody.name + "-" + selected.name +
-                 " Barycentric";
+          return Localizer.Format("#Principia_RFS_DynamicTitle_XYB",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:123]: Principia_ReferenceFrameSelector_Name_BarycentricRotating

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 27]
```diff
@@ -109,28 +110,32 @@ public enum FrameType {
...
+                                                selected.name);
         }
      case FrameType.BODY_CENTRED_PARENT_DIRECTION:
         if (selected.is_root()) {
           throw Log.Fatal(
               "Naming parent-direction rotating frame of root body");
         } else {
-          return selected.name + "-Centred " + selected.referenceBody.name +
-                 "-Aligned";
+          return Localizer.Format("#Principia_RFS_DynamicTitle_XCYA",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:132]: Principia_ReferenceFrameSelector_Name_BodyCentredParentDirection

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 28]
```diff
@@ -109,28 +110,32 @@ public enum FrameType {
...
+                                                 selected.name,
+                                   selected.referenceBody.name);
         }
      case FrameType.BODY_SURFACE:
-       return selected.name + "-Centred " + selected.name + "-Fixed";
+       return Localizer.Format("#Principia_RFS_DynamicTitle_XCXF",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:137]: Principia_ReferenceFrameSelector_Name_BodySurface

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 29]
```diff
@@ -140,27 +145,33 @@ public enum FrameType {
                                  CelestialBody selected,
                                  Vessel target_override) {
     if (target_override) {
-      return "Tgt LVLH@" + selected.name[0];
+      return Localizer.Format("#Principia_RFS_NavBallText_Target",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:148]: Principia_ReferenceFrameSelector_ShortName_Target, and same below in this function,
Principia_ReferenceFrameSelector_ShortName_BodyCentredNonRotating etc.

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 30]
```diff
@@ -170,38 +181,34 @@ public enum FrameType {
                                    CelestialBody selected,
                                    Vessel target_override) {
     if (target_override) {
-      return "Reference frame fixing the target vessel (" +
-             target_override.vesselName + "), the plane of its orbit around " +
-             selected.NameWithArticle() + ", and the line between them";
+      return Localizer.Format("#Principia_RFS_DynamicText_Target",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:184]: Principia_ReferenceFrameSelector_Description_Target, and same below in this function, Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating etc.

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 31]
```diff
@@ -219,15 +226,18 @@ public enum FrameType {
     if (!target_override &&
         (frame_type == FrameType.BODY_CENTRED_NON_ROTATING ||
          frame_type == FrameType.BODY_SURFACE)) {
-      return $"equator of {selected_celestial.NameWithArticle()}";
+      return Localizer.Format("#Principia_RFS_DynamicText_Plane", 
+                          selected_celestial.NameWithArticle());
     }
     string secondary = target_override != null
-        ? "the target"
+        ? Localizer.Format("#Principia_RFS_DynamicText_Sencondary")
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:233]: Principia_ReferenceFrameSelector_ReferencePlane_Secondary_Target

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 32]
```diff
@@ -219,15 +226,18 @@ public enum FrameType {
...
         : selected_celestial.NameWithArticle();
     string primary = target_override != null
         ? selected_celestial.NameWithArticle()
         : selected_celestial.referenceBody.NameWithArticle();
-    return $"orbit of {secondary} around {primary}";
+    return Localizer.Format("#Principia_RFS_DynamicText_PlaneReturn",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:238]: Principia_ReferenceFrameSelector_ReferencePlane

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 33]
```diff
@@ -274,8 +284,10 @@ public enum FrameType {
   }
 
   public void RenderButton() {
-    if (UnityEngine.GUILayout.Button(name_ + " selection (" + Name() +
-                                     ")...")) {
+    if (UnityEngine.GUILayout.Button(
+        Localizer.Format("#Principia_RFS_Button",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:288]: Principia_ReferenceFrameSelector_ToggleButton

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 34]
```diff
@@ -284,7 +296,9 @@ public enum FrameType {
   public CelestialBody selected_celestial { get; private set; }
   public Vessel target_override { get; set; }
 
-  protected override string Title => name_ + " selection (" + Name() + ")";
+  protected override string Title =>Localizer.Format("#Principia_RFS_Title",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:299]: Principia_ReferenceFrameSelector_Title

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 35]
```diff
@@ -297,9 +311,8 @@ public enum FrameType {
       using (new UnityEngine.GUILayout.VerticalScope()) {
         if (target_override) {
           UnityEngine.GUILayout.Label(
-              "Using target-centred frame selected on navball speed " +
-              "display\n\n" +
-              Description(frame_type, selected_celestial, target_override),
+              Localizer.Format("#Principia_RFS_DynamicText_EnableTarget",
```
Comment (eggrobin) [inline ksp_plugin_adapter/reference_frame_selector.cs:314]: Principia_ReferenceFrameSelector_TargetFrameSelected

**Review by eggrobin — CHANGES_REQUESTED** [review]
I have commented on all remaining identifiers; I will review the translations of technical terms once that is addressed.
Many of the technical translations need changes, but since the cfg file will completely change with the renamings, doing both at the same time would make it complicated to keep track of what has and hasn’t been done.

**Comment by pleroy** [comment]
retest this please

**Review by eggrobin — APPROVED** [review]

**Pull request merged**